docs/articles/documentation/using-testcafe/programming-interface/createtestcafe.md

```diff
@@ -9,20 +9,21 @@ checked: true
 Creates a [TestCafe](testcafe.md) server instance.
 
 ```text
-async createTestCafe([hostname], [port1], [port2]) → Promise<TestCafe>
+async createTestCafe([hostname], [port1], [port2], developmentMode) → Promise<TestCafe>
```

Comment:
The correct spelling  - `async createTestCafe([hostname], [port1], [port2], sslOptions, developmentMode) → Promise<TestCafe>`.